src/matbench_genmetrics/utils/match.py

```diff
@@ -148,5 +179,20 @@ def get_match_matrix(
             gen_structures,
             match_type="StructureMatcher",
             symmetric=symmetric,
+            verbose=verbose,
             **match_kwargs,
         )
+
+
+# %% Code Graveyard
+
+# if verbose:
+#     logger.info("Decorating structures with oxidation states")
+# oxi_structures = []
+# for s in structures:
+#     try:
+#         oxi_struct = bva.get_oxi_state_decorated_structure(s)
```

Comment:
yea, i never got this converged for real-world structures :D

Reply:
Oh, interesting!